lib/beaker/hypervisor/vagrant_virtualbox.rb

```diff
@@ -16,9 +16,13 @@ def self.vb_customize_vm(command, args)
   end
 
   def self.provider_vfile_section(host, options)
+    # Allow memory and CPUs to be set at a per node level or overall, and take the most specific setting
+    host_memory = host['vagrant_memsize'] ? host['vagrant_memsize'] : options['vagrant_memsize']
```

Comment:
If you are going to organize this all together, might as well roll in the default value here as well (ie, 1024 for memsize, 1 for num cpus).